bigquery/google/cloud/bigquery/client.py

```diff
@@ -1929,6 +1935,61 @@ def list_rows(
         )
         return row_iterator
 
+    def _schema_from_json_file_object(self, file):
+        """Helper function for schema_from_json that takes a
+           file object that describes a table schema.
+
+           Returns:
+                List of schema field objects.
+        """
+        schema_field_list = list()
+        json_data = json.load(file)
+
+        for field in json_data:
```

Comment:
FYI: This loop could be replaced with a Python "list comprehension".

Reply:
Changed to be a list comprehension.